pkg/usagestats/reporter_test.go

```diff
@@ -15,20 +15,15 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/grafana/loki/pkg/storage/chunk/local"
-	"github.com/grafana/loki/pkg/storage/chunk/storage"
 )
 
-var metrics = storage.NewClientMetrics()
-
 func Test_LeaderElection(t *testing.T) {
 	stabilityCheckInterval = 100 * time.Millisecond
 
 	result := make(chan *ClusterSeed, 10)
-	objectClient, err := storage.NewObjectClient(storage.StorageTypeFileSystem, storage.Config{
-		FSConfig: local.FSConfig{
-			Directory: t.TempDir(),
-		},
-	}, metrics)
+	objectClient, err := local.NewFSObjectClient(local.FSConfig{
```

Comment:
All the test changes in `usagestats` package were done to avoid a circular dependency.
It was caused by the import fix I made at https://github.com/grafana/loki/pull/5484/files#diff-8aa5c09f2d873a1f7c7a48d253da4d5d1a434cf2fa32ef84e5c1a68e0617f82aR15